presto-main/src/main/java/io/prestosql/eventlistener/EventListenerManager.java

```diff
@@ -55,59 +56,76 @@ public void addEventListenerFactory(EventListenerFactory eventListenerFactory)
         }
     }
 
-    public void loadConfiguredEventListener()
+    public void loadConfiguredEventListeners()
             throws Exception
     {
-        File configFile = CONFIG_FILE.getAbsoluteFile();
-        if (!configFile.exists()) {
+        if (!EVENT_LISTENER_CONFIGURATION_FILE.exists()) {
             return;
         }
-
-        Map<String, String> properties = new HashMap<>(loadProperties(configFile));
-
-        String name = properties.remove(NAME_PROPERTY);
-        checkState(!isNullOrEmpty(name), "Access control configuration %s does not contain '%s'", configFile, NAME_PROPERTY);
-
-        setConfiguredEventListener(name, properties);
+        Map<String, String> properties = new HashMap<>(loadProperties(EVENT_LISTENER_CONFIGURATION_FILE));
```

Comment:
Please model the loading code and logging after `AccessControlManager`

Reply:
I updated this to be more like the `AccessControlManager`